packages/core/src/domain/tracekit/computeStackTrace.ts

```diff
@@ -28,11 +29,14 @@ export function computeStackTrace(ex: unknown): StackTrace {
     stack,
   }
 }
-
+const url = '((?:file|https?|blob|chrome-extension|native|eval|webpack|<anonymous>|\\w+\\.|\\/).*?)'
+const position = '(?::(\\d+))'
 const CHROME_LINE_RE =
-  // eslint-disable-next-line max-len
-  /^\s*at (.*?) ?\(((?:file|https?|blob|chrome-extension|native|eval|webpack|<anonymous>|\/).*?)(?::(\d+))?(?::(\d+))?\)?\s*$/i
-const CHROME_EVAL_RE = /\((\S*)(?::(\d+))(?::(\d+))\)/
+  // eslint-disable-next-line local-rules/disallow-side-effects
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏You should not disable this rule, as it guarantees that our code will stay mostly tree-shakable. In this case, we have two possibilities:
* allow `new RegExp` by changing the rule [here](https://github.com/DataDog/browser-sdk/blob/dfcd3dc6a861be9a25d3b1cc533262096a687742/eslint-local-rules/disallowSideEffects.js#L197)
* build the RE in a function, like:
```ts
let chromeLineRe: RegExp | undefined
function getChromeLineRe() {
  if (!chromeLineRe) chromeLineRe = new RegExp(...)
  return chromeLineRe
}
```

We know that `new RegExp` is side-effect free, so the first solution could be good. One small drawback is that Webpack+Terser fails to treeshake regex constructed with variables like you do. This can be observed like that:
```
$ cat src/index.js
const a = new RegExp("foo");

const bar = "bar";
const b = new RegExp(`foo${bar}`);

$ npx --package webpack-cli webpack --mode production
assets by status 21 bytes [cached] 1 asset
./src/index.js 84 bytes [built] [code generated]
webpack 5.72.0 compiled successfully in 99 ms

$ cat dist/main.js
new RegExp("foobar");
```
`a` has been treeshaken in the resulting file, but `b` not entirely.

As this is a minor issue, and Terser might be improved someday, I suggest changing the rule nonetheless. (Note that [rollup does not have this issue](https://rollupjs.org/repl/?version=2.77.1&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmNvbnN0JTIwYSUyMCUzRCUyMG5ldyUyMFJlZ0V4cCglNUMlMjJmb28lNUMlMjIpJTNCJTVDbiU1Q25jb25zdCUyMGJhciUyMCUzRCUyMCU1QyUyMmJhciU1QyUyMiUzQiU1Q25jb25zdCUyMGIlMjAlM0QlMjBuZXclMjBSZWdFeHAoJTYwZm9vJTI0JTdCYmFyJTdEJTYwKSUzQiUyMiUyQyUyMmlzRW50cnklMjIlM0F0cnVlJTdEJTVEJTJDJTIyb3B0aW9ucyUyMiUzQSU3QiUyMmZvcm1hdCUyMiUzQSUyMmVzJTIyJTJDJTIybmFtZSUyMiUzQSUyMm15QnVuZGxlJTIyJTJDJTIyYW1kJTIyJTNBJTdCJTIyaWQlMjIlM0ElMjIlMjIlN0QlMkMlMjJnbG9iYWxzJTIyJTNBJTdCJTdEJTdEJTJDJTIyZXhhbXBsZSUyMiUzQW51bGwlN0Q=) 🎉  )